EIPS/eip-XXXX.md

```diff
@@ -0,0 +1,177 @@
+---
+Eip: XXXX *Number to be determined by Editor*
+Title: ERC Token in Multiple Languages
+Author: Jeremy Akatsa (@jeyakatsa)
+Status: Draft
+Type: Standards Track
+Category: ERC
+Created: 1/8/2022
+---
+
+## Simple Summary
+An ERC token/currency capable of being created in multiple languages (Java, Rust, Python, Go, etc).
+
+## Abstract
+While building the *[Light Client Infrastructure for the Teku Client (for Ethereum 2)](https://github.com/jeyakatsa/teku/tree/master/light-client),* a discovery was made; clients serve as arbiters or "bridges" to the Ethereum main chain.
+| ETH2 Client   | Language     | Team                |
+|:--------------|:------------ |:------------------- |
+| Teku          | Java         | Consensys           |
+| Lighthouse    | Rust         | Sigma Prime         |
+| Trinity       | Python       | Ethereum Foundation |
+| Prysm         | Go           | Prysmatic Labs      |
+
+Such clients offer differing languages that connect with the transactions offered on Ethereum, thus concluding if said clients can offer a base layer capable of inferring transactions from Ethereum's base layer, can tokens created in said languages also offer the same results?
+
```

Comment:
This section seems like content for the motivation section, rather than an abstract.

Instead, the abstract should be a technical summary of the EIP itself. Someone should be able to read only the abstract to get the gist of what this specification does.